practice/package.json

```diff
@@ -17,6 +17,7 @@
     "parcel": "^2.12.0"
   },
   "dependencies": {
-    "dotenv": "^16.4.5"
+    "dotenv": "^16.4.5",
+    "toastify-js": "^1.12.0"
```

Comment:
@JennyDoJD why should we need this package? Can you implement it by yourself?

Reply:
because I want to create a toast notification using toastify-js library and display it on UI.

Reply:
@JennyDoJD You are in js training course, you should do function/UI by yourself, void abuse lib because you don't know what is happening behind your feature. for now, pls go ahead, we will look back at it later